README.md

```diff
@@ -4,3 +4,84 @@
 [![Build Status](https://travis-ci.org/GitbookIO/slate-sugar.png?branch=master)](https://travis-ci.org/GitbookIO/slate-sugar)
 
 Set of Slate helpers to create nodes and documents.
+
+## Install
+
+```
+yarn add slate-sugar
+```
+
+## Motivation
```

Comment:
This part should be moved before the `Install` without a title (introduction of the module).